### PR TITLE
Try running fiat-crypto on large runners

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Comment by SkySkimmer** [comment]
@coqbot run full ci

**Comment by SkySkimmer** [comment]
Didn't work out.
Maybe leaving njobs at 2 would be less likely to hit memory limits

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -650,9 +650,12 @@ library:ci-fcsl_pcm:
   - library:ci-mathcomp_1
 
 library:ci-fiat_crypto:
-  extends: .ci-template-flambda
+  extends:
+    - .ci-template-flambda
+    - .large-tags
   variables:
     COQEXTRAFLAGS: "-async-proofs-tac-j 0"
+    NJOBS: "4"
```
Comment (SkySkimmer) [inline .gitlab-ci.yml:658]:
```suggestion
```

**Comment by SkySkimmer** [comment]
njobs=2 didn't work out either

@JasonGross what shall we do about fiat-crypto? timeouts are getting more frequent

**Comment by silene** [comment]
Is the `large` tag even honored by the custom runners?

**Comment by SkySkimmer** [comment]
cc @thierry-martinez

**Comment by Zimmi48** [comment]
Maybe worth opening an issue at https://gitlab.inria.fr/inria-ci/custom-runner/?

**Comment by ppedrot** [comment]
Since the fiat-crypto issue has been worked around and this can be considered an experiment, I'm closing the PR.

**Comment by SkySkimmer** [comment]
fiat-crypto still has issues with out of memory, they're just less common now AFAIK

**Comment by SkySkimmer** [comment]
TODO try again after https://gitlab.inria.fr/inria-ci/custom-runner/-/merge_requests/11 is merged

**Comment by SkySkimmer** [comment]
@coqbot run full ci

**Comment by SkySkimmer** [comment]
We stopped using the custom runner due to https://gitlab.inria.fr/inria-ci/custom-runner/-/issues/28 and although we have fewer runners now they never (AFAICT) have runner issues so we probably won't be going back.

**Pull request closed without merging**